examples/basic/app.R

```diff
@@ -2,13 +2,15 @@ library(shiny)
 library(shiny.i18n)
 
 # File with translations
-i18n <- Translator$new(translation_json_path = "../data/translation.json")
+i18n <- Translator$new(translation_json_path = "../data/translation.json", js = TRUE)
 
 # Change this to en
-i18n$set_translation_language("pl")
+i18n$set_translation_language("en")
```

Comment:
switch to pl or remove the comment above

Reply:
done